packages/core/src/i18n.test.ts

```diff
@@ -331,4 +331,27 @@ describe("I18n", () => {
       expect(missing).toHaveBeenCalledWith("en", "missing")
     })
   })
+
+  it("._ should parse unicode sequences", () => {
+    const messages = {
+      "Software development": "Software\\u00ADentwicklung",
+    }
+    const i18n = setupI18n({
+      locale: "de",
+      messages: { de: messages },
+    })
+    expect(i18n._("Software development")).toEqual("Software­entwicklung")
+  })
```

Comment:
```suggestion
```

Reply:
@vonovak Thank you for the review! I wanted to show that before the fix, the first test passed, but the second failed. I agree it can be leaved only the one that failed without the fix. Did it in [d1efca4](https://github.com/lingui/js-lingui/pull/2030/commits/d1efca43f695ea563fb4ad0656bef40606ec704f).